lumen/ai/coordinator.py

```diff
@@ -385,38 +389,34 @@ async def _execute_graph_node(self, node: ExecutionNode, messages: list[Message]
                         f"Avoid doing the same analysis as any of these custom analyses: {custom_analysis_doc} "
                         f"Most likely, you'll just need to do a simple SELECT * FROM {{table}};"
                     )
-                    custom_messages.append({"role": "user", "content": custom_message})
-            if instruction:
```

Comment:
Dropped this because I already include the full plan
https://github.com/holoviz/lumen/pull/844/commits/6bfbc2a31c6d0af7cba321f3c2b58f7dda58f9a1#diff-d99fd924b197130083f3cc0db28819f92a929aab5aef9f8d17a72b5a9aa7fbf5R683